bot.py

```diff
@@ -13,22 +14,18 @@
 awake = {}
 
 def readTimesFromFile():
+    servers = ServerDao.get_all_servers()
+
     global serverAndDate
-    with open("timeStamps.txt", "r") as target:
-        for line in target:
-            tmp = line.split(',')
-            tmp[1] = tmp[1][0:-1]
-            serverAndDate[tmp[0]] = datetime.datetime.strptime(tmp[1], "%Y-%m-%d %H:%M:%S")
-            if (len(tmp) >= 3):
-                awake[tmp[0]] = bool(tmp[2])
-            else:
-                awake[tmp[0]] = True
+    for server in servers:
+        serverAndDate[server[0]] = datetime.datetime.strptime(server[1], "%Y-%m-%d %H:%M:%S")
+        awake[server[0]] = bool(server[2])
             
 
 def writeTimesToFile():
     with open('timeStamps.txt', 'w') as target:
```

Comment:
I get that I'm going to be refactoring this whole flow to make proper use of this change anyway, but this file open is useless.